.github/actions/tests/run-e2e-tests/action.yml

```diff
@@ -105,6 +105,49 @@ runs:
           eval "$pytest_cmd"
         fi
 
+    - name: Handle Debug Logs
+      if: always()
+      shell: bash
+      run: |
+        # Determine test phase based on keyword
+        if [[ "${{ inputs.keyword }}" == "test_get_status or test_get_params" ]]; then
+          TEST_PHASE="smoke"
+        elif [[ "${{ inputs.tests }}" == "postmerge" ]]; then
+          if [[ "${{ inputs.mc_epoch }}" == "3" ]]; then
+            TEST_PHASE="postmerge-final"
+          else
+            TEST_PHASE="postmerge-epoch${{ inputs.mc_epoch }}"
+          fi
+        else
+          TEST_PHASE="premerge"
+        fi
```

Comment:
This won't scale well when we add/change test suites. Debug logs are used when something fails, so let's just use the same name for every artifact and overwrite